src/higley_lab_to_nwb/lohani_2022/lohani_2022nwbconverter.py

```diff
@@ -24,19 +24,16 @@ class Lohani2022NWBConverter(NWBConverter):
 
     def __init__(
         self,
-        excitation_types: List[str],
-        channels: List[str],
+        excitation_type_channel_comb: dict,
```

Comment:
"combination" is not that longer, and it improves clarity
```suggestion
        excitation_type_channel_combination: dict,
```